tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -987,3 +987,12 @@ vrf/test_vrf_attr.py::TestVrfAttrSrcMac::test_vrf1_neigh_with_default_router_mac
     reason: "RIF MAC taking precedence over VRF MAC"
     conditions:
       - "asic_type in ['barefoot']"
+
+#######################################
+#####          cutsom_acl         #####
+#######################################
+acl/custom_acl_table/test_custom_acl_table.py:
```

Comment:
@bingwang-ms One minor comment, could you please move this line ahead of this file and keep all sections in alphabetical order.
Here Yutong has this requirement in her PR:
https://github.com/sonic-net/sonic-mgmt/pull/5114

Reply:
Fixed. Thanks